src/c++/perf_analyzer/test_load_manager_base.h

```diff
@@ -36,6 +36,49 @@ namespace cb = triton::perfanalyzer::clientbackend;
 
 namespace triton { namespace perfanalyzer {
 
+class InferContextMockedInferInput : public InferContext {
```

Comment:
Why not call this MockInferContext?

Reply:
If you want to support both mocking and not mocking the infer input, perhaps we can have a bool for the constructor